trollmoves/server.py

```diff
@@ -380,19 +382,34 @@ def create_file_notifier(attrs, publisher):
 
     tmask = (pyinotify.IN_CLOSE_WRITE |
              pyinotify.IN_MOVED_TO |
-             pyinotify.IN_CREATE)
+             pyinotify.IN_CREATE |
+             pyinotify.IN_DELETE)
 
     wm_ = pyinotify.WatchManager()
 
     pattern = globify(attrs["origin"])
     opath = os.path.dirname(pattern)
 
+    if 'origin_inotify_base_dir_skip_levels' in attrs:
```

Comment:
A short comment here would be nice what's been done.

Reply:
OK